spec/integ/crypto/verification.spec.ts

```diff
@@ -1338,7 +1383,7 @@ describe("verification", () => {
         async function sendBackupGossipAndExpectVersion(
             requestId: string,
             secret: string,
-            expectBackup: KeyBackupInfo,
+            expectBackup?: KeyBackupInfo | { errcode?: string; error?: string },
```

Comment:
could you document the behavior of `expectBackup`? What does it mean if it is `undefined`, what if it has an `errcode`, etc?